### ISSUE TITLE
No way of knowing which files are not in the resulting list from a scan...

### ISSUE THREAD

**Issue by chinagreenelvis** [libretro/RetroArch#13312]
## Description

Scanning a directory of ROM files creates a list with significantly fewer games than the directory contains. Without going through the list one-by-one, there is no way to know which files were not added.

### Expected behavior

A logfile of some kind that produces a list of every file that did not get added to the list. An optional filter for recognized filetypes would also help.

Additionally, a list of missing ROMs?

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.9.12

**Comment by kbdharun** [comment]
Yeah, This Feature will be very useful, to identify non-compatible ROMs with Retroarch 

### **Version**
 

- RetroArch: 1.9.13

**Comment by chinagreenelvis** [comment]
It's the only way a user has any personal control.

**Comment by Justme488** [comment]
@chinagreenelvis you should edit the title to include [Feature Request].
